api/src/pcapi/alembic/versions/20231005T070649_9d9bfc829be5_remove_clg_5_6_ff.py

```diff
@@ -0,0 +1,33 @@
+"""remove WIP_ADD_CLG_6_5_COLLECTIVE_OFFER ff"""
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "9d9bfc829be5"
+down_revision = "06ebfa9392fa"
+branch_labels = None
+depends_on = None
+
+
+def get_flag():  # type: ignore[no-untyped-def]
+    # Do not import `pcapi.models.feature` at module-level. It breaks
+    # `alembic history` with a SQLAlchemy error that complains about
+    # an unknown table name while initializing the ORM mapper.
+    from pcapi.models import feature
+
+    return feature.Feature(
+        name="WIP_ADD_CLG_6_5_COLLECTIVE_OFFER",
+        isActive=False,
```

Comment:
Le feature flag est actuellement activé. Si on doit le remettre (dans le "downgrade"), on doit le remettre à la valeur actuelle.

```suggestion
        isActive=True,
```

Reply:
Ah yes merci !